.github/workflows/codeql.yml

```diff
@@ -0,0 +1,82 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
```

Comment:
I guess we could use the same list as for the other workflows; https://github.com/docker/cli/blob/5fc42fc64e7b0d96dc04233efaece63ea232e8d4/.github/workflows/e2e.yml#L8-L14

Reply:
Will do, thanks!